test/bin/run-tests-docker.sh

```diff
@@ -4,34 +4,80 @@ set -e
 unset CDPATH
 cd "$( dirname "${BASH_SOURCE[0]}" )/../.."
 
-IMAGE_NAME_SCITRAN_CORE=scitran-core
+IMAGE_NAME_SCITRAN_CORE="scitran-core:run-tests"
 IMAGE_NAME_MONGO=mongo
-CONTAINER_NAME_MONGO=some-mongo
+CONTAINER_NAME_MONGO=scitran-core-test-mongo
+CONTAINER_NAME_SCITRAN_CORE=scitran-core-test-uwsgi
+
+
+USAGE="
+    Run scitran-core tests using docker
+\n
+    Usage:\n
+    \n
+    --help: print help and exit\n
+    -b, --build-image: Rebuild scitran-core base image\n
+    -L, --no-lint: Skip linter\n
+
+"
+
+SCITRAN_RUN_LINT="true"
+BUILD_IMAGE="false"
+
+while [ "$#" -gt 0 ]; do
+    key="$1"
+    case $key in
+        --help)
+        echo -e $USAGE >&2
+        exit 1
+        ;;
+        -b|--build-image)
+        BUILD_IMAGE="true"
+        ;;
+        -L|--no-lint)
+        SCITRAN_RUN_LINT="false"
+        ;;
+        *)
+        echo "Invalid option: $key" >&2
+        echo -e $USAGE >&2
+        exit 1
+        ;;
+    esac
+    shift
+done
+
 
 clean_up () {
-  # Stop and remove mongo container
+  # Copy coverage file to host for possible further reporting
+  docker cp "$CONTAINER_NAME_SCITRAN_CORE":/var/scitran/code/api/.coverage .coverage || true
+  # Stop and remove containers
   docker rm -v -f "$CONTAINER_NAME_MONGO"
+  docker rm -v -f "$CONTAINER_NAME_SCITRAN_CORE"
 }
 trap clean_up EXIT
 
+if [[ $( docker images "$IMAGE_NAME_SCITRAN_CORE" | tail -n +2 ) == "" ]]; then
+  echo "scitran-core image not found.  Building"
```

Comment:
Change `scitran-core` here to use `$IMAGE_NAME_SCITRAN_CORE` so the message is more explicit. Sorry I missed this on my first review.